lib/src/parse/scss.dart

```diff
@@ -158,8 +158,11 @@ class ScssParser extends StylesheetParser {
           scanner.spanFrom(start));
     }
 
-    return SilentComment(
+    var silentComment = SilentComment(
         scanner.substring(start.position), scanner.spanFrom(start));
+
+    lastSilentComment = silentComment;
+    return silentComment;
```

Comment:
Don't forget to add this logic to the indented syntax parser as well.

Reply:
Done.